xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -397,8 +397,11 @@ function displayUsersAndGroups(row, i, table, idx, form_token, targetDocument)
 {
   var userurl = row.userurl;
   var uorg = table.json.uorg;
-  var allows = row.allows;
-  var denys = row.denys;
+  // Rights are case insensitive in XWiki security module, and the reference rights defined in rightsUI.vm
+  // are lowercase, hence we make sure to convert all applying rights to lowercase before comparing them with the
+  // reference ones (see XWIKI-18556).
```

Comment:
I try to avoid reference to jira issue in the code and instead make sure that everything needed is in the comment (which seems to be the case here).

Reply:
Thank you Thomas for the review. I removed the jira reference indeed.